src/main/java/rife/instrument/RifeTransformer.java

```diff
@@ -69,5 +67,5 @@ public final byte[] transform(ClassLoader loader, String className, Class<?> cla
      * @since 1.0
      */
     protected abstract byte[] transformRife(ClassLoader loader, String classNameInternal, Class<?> classBeingRedefined, ProtectionDomain protectionDomain, byte[] classfileBuffer)
-    throws IllegalClassFormatException;
```

Comment:
I think this needs to stay there in case other `RifeTransformer` implementations need to throw this exception.